src/plugins/shutdown_button/ShutdownButton.cc

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include "ShutdownButton.hh"
+
+#include <csignal>
+#include <string>
+
+#include <ignition/common/Console.hh>
+#include <ignition/common/StringUtils.hh>
+#include <ignition/common/Time.hh>
```

Comment:
Is `Time.hh` needed? That's deprecated on `ign-common4` and removed on `ign-common5` in favor of `chrono`.